_posts/2017-01-23-upgrade-guide-for-babel-7-authors.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title:  "Upgrade Guide for Babel 7 (Tool Authors)"
+author: Sven SAULEAU
+date:   2017-01-23 12:00:00
+categories: announcements
+share_text: "Upgrade Guide for Babel 7 (Tool Authors)"
+third_party_js:
+- https://platform.twitter.com/widgets.js
+custom_js_with_timestamps:
+- docs.js
+---
+
+This document is for developers that create tools that depend on Babel such as babel plugins and the relevant changes for upgrading to Babel 7.
+
+> Also check out the [User Upgrade Guide](/blog/2017/01/23/upgrade-guide-for-babel-7) for other relevant changes.
+
+## All Babel packages
+
+> Support for Node.js 0.10 and 0.12 has been dropped
+
+> Likelihood to break your CI: high
```

Comment:
yeah something like that - should we even have a none? dono what counts as none (class constructor call? it's breaking so technically it's a risk)